src/app/profile/payment-methods/payment-method/payment-method.component.js

```diff
@@ -0,0 +1,34 @@
+import angular from 'angular';
+import template from './payment-method.tpl';
+
+class PaymentMethodController{
+
+  /* @ngInject */
+  constructor(){
+    this.isCollapsed = true;
+  }
+
+  getExpiration() {
+    return `EXPIRES ${this.model.expiry_month}/${this.model.expiry_year}`;
+  }
+
+  getLastFourDigits(){
+    let cardNumber = this.model.card_number;
+    return `ending in ****${cardNumber.substr(cardNumber.length-4,this.model.card_number.length-1)}`;
```

Comment:
I know you might not have seen this in the api yet but the api should only give you the last 4 digits or already have prefixed a bunch of starts to it. I think I've seen it both ways. If it ever gives you a full number, I'd make that known since that's a security issue. Again I think the `ending in` belongs in the view so it can be translated.